constantine/build.gradle

```diff
@@ -39,17 +39,36 @@ if (osName.contains('mac') && osArch.contains('aarch64')) {
     libDir = 'linux-gnu-x86_64'
 }
 
-task libCopy(type: Copy) {
-    from "build/${libDir}/lib/"
-    into "build/resources/main/lib/${libDir}"
+task macArmLibCopy(type: Copy) {
+    from "build/darwin-aarch64/lib/libconstantine.dylib"
+    from "build/darwin-aarch64/lib/libconstantineeip196.jnilib"
+    into 'build/resources/main/lib/darwin-aarch64'
```

Comment:
FYI, testling locally I am getting
`Exception in thread "main" java.lang.UnsatisfiedLinkError: no constantineeip196 in java.library.path: ...`

Reply:
Which OS are you testing this in?
I assume first you ran the ./build.sh and then the test files?

I tested in Mac and Linux x86_64, working fine for me

Reply:
Which OS are you testing this in?
I assume first you ran the ./build.sh and then the test files?

I tested in Mac and Linux x86_64, working fine for me

Reply:
I am testing by integrating into besu on Darwin aarch64, when I build I am seeing the libs getting embedded in `lib/{arch}` rather than `{arch}`.  The 0.9.4 build shows the same.  

Perhaps there is a classpath difference / expectation in besu-native/constantine that allows it to find the libs, but that same native classpath isn't in besu (since besu is using the jar dep instead of a filesystem build artifact).  Looking into it now